src/main/java/org/apache/sysml/utils/lite/BuildLiteExecution.java

```diff
@@ -0,0 +1,696 @@
+			invokeMethod(clazz, obj, "testInputStreamReadFrameDenseCSV");
+			invokeMethod(clazz, obj, "testInputStreamReadFrameDenseText");
+			invokeMethod(clazz, obj, "testInputStreamReadFrameSparseCSV");
+			invokeMethod(clazz, obj, "testInputStreamReadFrameSparseText");
+			invokeMethod(clazz, obj, "testInputStreamReadFrameDenseCSVMeta");
+			invokeMethod(clazz, obj, "testInputStreamReadFrameDenseTextMeta");
+			invokeMethod(clazz, obj, "testInputStreamReadFrameSparseCSVMeta");
+			invokeMethod(clazz, obj, "testInputStreamReadFrameSparseTextMeta");
+			invokeMethod(clazz, obj, "tearDown");
+
+			clazz = cl.loadClass("org.apache.sysml.test.integration.functions.jmlc.MulticlassSVMScoreTest");
+			obj = clazz.newInstance();
+			invokeMethod(clazz, obj, "setUpBase");
+			invokeMethod(clazz, obj, "setUp");
+			invokeMethod(clazz, obj, "testJMLCMulticlassScoreDense");
+			invokeMethod(clazz, obj, "testJMLCMulticlassScoreSparse");
+			invokeMethod(clazz, obj, "tearDown");
+
+			clazz = cl.loadClass("org.apache.sysml.test.integration.functions.jmlc.ReuseModelVariablesTest");
+			obj = clazz.newInstance();
+			invokeMethod(clazz, obj, "setUpBase");
+			invokeMethod(clazz, obj, "setUp");
+			invokeMethod(clazz, obj, "testJMLCScoreGLMDense");
+			invokeMethod(clazz, obj, "testJMLCScoreGLMSparse");
+			invokeMethod(clazz, obj, "testJMLCScoreGLMDenseReuse");
+			invokeMethod(clazz, obj, "testJMLCScoreGLMSparseReuse");
+			invokeMethod(clazz, obj, "testJMLCScoreMSVMDense");
+			invokeMethod(clazz, obj, "testJMLCScoreMSVMSparse");
+			invokeMethod(clazz, obj, "testJMLCScoreMSVMDenseReuse");
+			invokeMethod(clazz, obj, "testJMLCScoreMSVMSparseReuse");
+			invokeMethod(clazz, obj, "tearDown");
+
+		} catch (Exception e) {
+			e.printStackTrace();
+		}
+	}
+
+	@SuppressWarnings({ "unchecked", "rawtypes" })
+	private static void invokeMethod(Class clazz, Object obj, String methodName) throws NoSuchMethodException,
+			SecurityException, IllegalAccessException, IllegalArgumentException, InvocationTargetException {
+		Method m = clazz.getMethod(methodName);
+		m.invoke(obj);
+	}
+
+	public static double[][] matrix(int rows, int cols, double[] vals) {
+		double[][] matrix = new double[rows][cols];
+		if ((vals == null) || (vals.length == 0)) {
+			return matrix;
+		}
+		for (int i = 0; i < vals.length; i++) {
+			matrix[i / cols][i % cols] = vals[i];
+		}
+		return matrix;
+	}
+
+	public static double[][] randomMatrix(int rows, int cols, double min, double max, double sparsity) {
+		double[][] matrix = new double[rows][cols];
+		Random random = new Random(System.currentTimeMillis());
+		for (int i = 0; i < rows; i++) {
+			for (int j = 0; j < cols; j++) {
+				if (random.nextDouble() > sparsity) {
+					continue;
+				}
+				matrix[i][j] = (random.nextDouble() * (max - min) + min);
+			}
+		}
+		return matrix;
+	}
+
+	public static void displayMatrix(double[][] matrix) {
+		System.out.println("Matrix size:" + matrix.length + "x" + matrix[0].length);
```

Comment:
you might want to replace this with DataConverter.toString(DataConverter.convertToMatrixBlock(matrix)).